code/app/src/main/java/com/adobe/marketing/mobile/consentTestApp/ConsentTestApplication.java

```diff
@@ -44,16 +41,10 @@ public void onCreate() {
 		*/
         MobileCore.configureWithAppID(LAUNCH_ENVIRONMENT_ID);
 
-        // register Adobe core extensions
-        try {
-            Identity.registerExtension();
-            Signal.registerExtension();
-            Consent.registerExtension();
-            Edge.registerExtension();
-            Assurance.registerExtension();
-        } catch (InvalidInitException e) {
-            e.printStackTrace();
-        }
+        // register AEP Mobile extensions
+        Consent.registerExtension();
+        Edge.registerExtension();
```

Comment:
Should we also include EdgeIdentity for Edge to work?

Reply:
good point, though they both need to be published to be available for this test app.